virtual-setup/slack-procedures.md

```diff
@@ -1 +1,115 @@
+
+**Responses to questions should be in [threads](https://slack.com/help/articles/115000769927-Use-threads-to-organize-discussions-) as much as possible**, including any screenshots or images. 
+To start a thread, hover over the message you want to respond to and click the start a thread icon that appears on the top right of the message:
+
+<img src = screenshots/slack-start-a-thread.png width = 300>
+
+You can then enter your response in the thread sidebar that appears on the right side of the Slack interface. 
+In general, you do not want to check the box that sends the message to the entire channel.
+
+#### Collecting feedback
+
+We also use the training-specific private channel to poll participants via [Polly](https://www.polly.ai/slack-poll). We collect the following feedback anonymously:
+
+* The most difficult or confusing point of the module ("muddiest point"), which is posted anonymously to the channel
+* What you liked about the module (anonymous, only instructors will see your answer)
+* How we can improve the module (anonymous, only instructors will see your answer)
+
+We will also ask you what you want to work on the next day on days _preceding_ consultation days.
+Your answers will be sent to course instructors only and are **not** anonymous.
+This allows instructors to group course participants based on their interests.
+
+Use the `Submit Response` button in a Polly message to enter your feedback:
+
+<img src = screenshots/slack-polly-response.png width = 500>
+
+### Using Slack calling during training
+
+If you need help during instruction or on consultation days and it is helpful to share your screen, a course instructor will initiate a [Slack call](https://slack.com/help/articles/216771908-Make-calls-in-Slack).
+Your instructor will ask you to [share your screen](https://slack.com/help/articles/115003501303-Share-your-screen-with-Slack-Calls) using the share your screen icon circled in yellow below.
+
+<img src = screenshots/slack-call-control.png width = 300>
+
+_Note: Where possible please utilize headphones throughout the training workshop to minimize audio feedback._
+
+### Using direct messages during training
+
+If you have a question that is _highly specific_ to your own data, you may [direct message](https://slack.com/help/articles/212281468-What-is-a-direct-message) a CCDL staff member. 
+
+First, use the new message button in the top right side corner of the Slack interface.
+
+<img src = screenshots/slack-compose-new-message.png width=300>
+
+You are then able to search for the CCDL instructors – `Chante Bethell`, `Candace Savonen`, `Josh Shapiro` or `Jaclyn N. Taroni` – and compose your message.
+
+Course instructors may direct you to the training-specific channel for more general questions or to another instructor where appropriate.
+
+## General Slack use
+
+In this section, we include a collection of tips that you may find helpful for using throughout training that were not covered above.
+
```

Comment:
I'm not sure if the formatting bar is visible by default. Do you think we should mention how to toggle it visible if they do not see it?

Reply:
How _does_ one toggle it if it's not visible? I don't know how to do that.

Reply:
Oh you hit the `Aa`
![2020-04-15 10 55 42](https://user-images.githubusercontent.com/15315514/79352352-b9fbb180-7f07-11ea-8c58-c54448e65fd0.gif)